### DISCUSSION TITLE
Problems with Android System WebView on Samsung devices

### DISCUSSION THREAD

**Discussion by MedaiP90** [ionic-team/capacitor#4055] [Q&A] [answered]
Does anyone has seen app crashes caused by the **Android System WebView** errors on *Samsung devices* (expecially newer ones: *Android 9* or *10*)? Some users of an app my company made experienced this, but I failed reproducing the problem and cannot find very much information on the internet about similar problems.

**Comment by BroadBrander** [comment]
I test on Samsung (Android 10) never happened.

**Comment by phal0r** [comment]
I can see those crashes in Google Play too, especially on new Samsung Devices (Android 9-11). The error is so unspecific, that there is no clue, where to dig deeper.

**Comment by MedaiP90** [comment, marked as answer]
I've found a solution:

- disable Google Chrome from device settings
- update Android System WebView from the Google Play Store
- (verify that your app is now working, it should)
- re-enable Google Chrome from device settings
- (update Google Chrome also if needed)

I hope this will be usefull to others.

**Comment by danbarretto** [comment]
I found a solution for Capacitor 4 Ionic 6 on a galaxy S9 using android 10 and webview 81. The problem seems to be related to the css processing when using the --prod flag. Check out this post:
https://forum.ionicframework.com/t/ionic-6-default-project-white-screen-on-android-30-or-below-when-using-prod/228087/3

After applying the following css class to my variables.scss, the home screen appeared :)
`.ion-page {
    padding: 0;
    width: 100%;
    height: 100%;
}`
